scripts/cloudcore_node.sh

```diff
@@ -0,0 +1,106 @@
+#! /bin/bash
+
+set -e
+
+# Enter the IP address of the Machine-1: Cloud Core Node, Machine-2: Edge Node with Control Plane, Machine-3: Edge Worker node 
+echo "Enter IP ADDRESS of Machine 1:"
+read ip_m1 
+echo "Enter IP ADDRESS of Machine 2:"
+read ip_m2
+echo "Enter ROOT password of Machine 2 for copying the CA, CERTS and Kubeconfig files from Machine 1"
+read -s pass_2
+echo "Enter IP ADDRESS of Machine 3:"
+read ip_m3
+echo "Enter ROOT password of Machine 3 for copying the CA, CERTS files to Machine 3"
+read -s pass_3
+#To kill running process of cloudcore
+cloudcore=`ps -aef | grep _output/local/bin/cloudcore | grep -v sh| grep -v grep| awk '{print $2}'`
+DIR=$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )
+if [ -f "${DIR}/prerequisite_packages.sh" ]; then
+source "${DIR}/prerequisite_packages.sh"
+fi
+
+pushd $HOME
+echo '*****SETTING UP THE HOSTNAME NODE-A*****'
+sudo hostnamectl set-hostname node-a
+echo '*****DISABLING FIREWALL*****'
+sudo ufw disable
+sudo swapoff -a
+
+key_gen (){
+   if [ "$(ls /root/.ssh/id_rsa.pub)" != "/root/.ssh/id_rsa.pub" ]
+   then
+      echo y | apt-get update
+      echo y | apt-get install sshpass
+      < /dev/zero ssh-keygen -q -N ""
+      sshpass -p $pass_2 ssh-copy-id -o StrictHostKeyChecking=no root@$ip_m2
+      sshpass -p $pass_3 ssh-copy-id -o StrictHostKeyChecking=no root@$ip_m3
```

Comment:
See reference doc: http://www.linuxproblem.org/art_9.html

See if use "a@A:~> cat .ssh/id_rsa.pub | ssh b@B 'cat >> .ssh/authorized_keys'
b@B's password: "
copy  ".ssh/id_rsa.pub" append to " .ssh/authorized_keys"
![ssh login without password](https://user-images.githubusercontent.com/90352709/153304928-07c423d7-e3d5-4ddc-98d9-ccc06e42b20f.png)